src/components/layout/editor-components/widget-editors/linear-layout-editor.tsx

```diff
@@ -73,6 +74,23 @@ class LinearLayoutEditor extends React.Component<Props, State> {
             onSelectChange={ this.props.onValueChange }
             selectItemType={ LinearLayoutOrientationValues }
           />
+        <HelpDialog title={ strings.layoutEditor.linearLayout.orientation }>
+          <Typography>
+            { strings.helpDialogs.layoutEditor.linearLayout.introduction }
+          </Typography>
+          <Box mt={ 2 }>
+            <Typography variant="h5">Vertical</Typography>
```

Comment:
This probably can be localized?
```suggestion
            <Typography variant="h5">
              Vertical
            </Typography>
```